traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityCalendarResultViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  AvailabilityCalendarResultViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-10-28.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+private let edgeInset = 10
```

Comment:
This is not an inset. Inset is its own type

Also these values dont belong here,...they need to be encapsulated inside a custom UIView and be configurable